sql/moz-fx-data-shared-prod/search_derived/desktop_mobile_monthly_search_aggregates_v1/query.sql

```diff
@@ -0,0 +1,60 @@
+-- Query for search_derived.desktop_mobile_monthly_search_aggregates_v1
```

Comment:
Nit: I'd call this something like `desktop_mobile_search_clients_monthly` since it's per-client per-month. The "aggregates" language I think is used elsewhere for non-client level data